docs/architecture/modernize-desktop/example-migration-core.md

```diff
@@ -0,0 +1,272 @@
+
+Now you can update the .csproj file. You'll delete the entire content and replace it with the following code:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+    <PropertyGroup>
+        <OutputType>WinExe</OutputType>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <UseWindowsForms>true</UseWindowsForms>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    </PropertyGroup>
+</Project>
+```
+
+Save and reload the project. You're now done updating the project file and the project is targeting the .NET Core.
+
+If you compile the project at this point, you'll find some errors related to the WCF client reference. Since this code is autogenerated, you must regenerate it to target .NET Core.
+
+![Screenshot of compilation errors on Visual Studio](./media/example-migration-core/winforms-compilation-errors.png)
+
+Delete the *Reference.cs* file and generate a new Service Client.
+
+Right-click over *Connected Services* and select the "*Add Connected Service*"
+option.
+
+![Screenshot of the Connected Services menu with the Add Connected Service option selected](./media/example-migration-core/add-connected-service.png)
+
+The Connected Services window opens. Select the **Microsoft WCF Web Service** option.
+
+![Screenshot of the Connected Services window](./media/example-migration-core/connected-services-window.png)
+
+If you have the WCF Service in the same solution as we have in this example, you can select the **Discover** option instead of specifying a service URL.
+
+![Screenshot of the Configure WCF Web Service Reference window](./media/example-migration-core/configure-wcf-reference.png)
+
+Once the service is located, the tool reflects the API contract implemented by the service. Change the name of the namespace to be `eShopServiceReference` as shown in the following image:
+
+![Screenshot the API contract and the namespace changed](./media/example-migration-core/api-contract-namespace.png)
+
+Select the **Finish** button. After a while, you'll see the generated code.
+
+You should see three autogenerated files:
+
+1. Getting Started: a link to GitHub to provide some information on WCF.
+2. *ConnectedService.json*: configuration parameters to connect to the service.
+3. *Reference.cs*: the actual WCF client code.
+
+![Screenshot of the Solution Explorer window with the three autogenerated files](./media/example-migration-core/autogenerated-files.png)
+
+If you compile again, you'll see many errors coming from *.cs* files inside the *Helper* folder. This folder was present in the .NET Framework version but not included in the old .csproj. But with the new SDK-style project, every code file present underneath the project file location is included by default. That is, the new .NET Core project tries to compile the files inside the *Helper* folder. Since that folder isn't needed, you can safely delete it.
```

Comment:
```suggestion
If you compile again, you'll see many errors coming from *.cs* files inside the *Helper* folder. This folder was present in the .NET Framework version but not included in the old *.csproj*. But with the new SDK-style project, every code file present underneath the project file location is included by default. That is, the new .NET Core project tries to compile the files inside the *Helper* folder. Since that folder isn't needed, you can safely delete it.
```